goal_time_1: "12:41"
goal_time_2: "12:42"
goal_time_3: "12:44"
goal_time_4: "12:46"
goal_time_5: "12:48"
